pqcrypto-template/scheme/README.md.j2

```diff
@@ -28,10 +28,10 @@ methods only.
 {% for scheme in schemes %}
 * ``{{ scheme.name }}``
 {% for implementation in scheme.implementations %}
-{% if implementation == 'avx2' or implementation == 'avx' or implementation == 'aesni' %}
-  * ``{{ implementation }}`` (if supported)
+{% if implementation == 'clean' %}
+  * ``{{ implementation }}`` (default)
 {% else %}
-  * ``{{ implementation }}``
+  * ``{{ implementation }}`` (if supported)
```

Comment:
There's schemes with alternate implementations where it's not just based on feature detection (notably FrodoKEM)

Reply:
The feature detection is rolled back to the previous logic and the "AArch64" option is added.